### PR TITLE
Feature/career page

### PR DESCRIPTION
**Pull request by katedee**
New career page/ #2124 
Refer[ to the figma file](https://www.figma.com/file/LW1JsT0h3IY2trMHdIVDQc/CDS-Website-Revamp-(Work-in-Progress)?node-id=132%3A8007)
A couple things to note/things to do:

- I left the image in and blended it with the new design, as I felt the page could use some visual breaking up, and until we decide on some illustrations or something, let’s keep all those lovely faces :)
- Heading order should be better now (thanks for the draft spot @CalvinRodo)
- Things missing from this page, but that I suspect need to be done through Strappi (and thus, another repo):
    - Updating the page title
    - Job postings will need a new category for section (development, design, etc). We’ll need to add that via strappi and then display on the page.

For other tasks/out of scope here:
- updating that page header with the titles, to match designs. We'll need this to be more controlled at the page level, or have some way to pass the appropriate background colour.
- 
image

localhost_1314_rejoindre-notre-equipe_ (2)

### CODE REVIEW

**Comment by CalvinRodo** [comment]
I ran this through the [WAVE browser add-on](https://wave.webaim.org/extension/) to take a look at the page structure in particular the headings 

While the heading orders are in the correct order and we have no h3's under h4's unfortunately we have a bit of what I would consider a tiny issue with them.

If you look at the heading structure you'll see that "At CDS, we welcome and invite difference" is underneath "Privacy Policy" I'm not sure that's correct, headings should make sense in regards to the structure of the page and I'm pretty sure we don't mean to include that heading in our privacy policy section. 

image

File: content/en/join-our-team/_index.html
Comment (CalvinRodo) [inline content/en/join-our-team/_index.html:46]: Usually for svg's you'd add a role="img" so that screen readers don't read out all the inner elements, but since this is for presentation you should probably add role="presentation" but axe isn't complaining so /shrug

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
See my comment about the semantic structure of the page. 

There is also an issue with the `nav` tags but that will be fixed when merging from main

File: content/en/join-our-team/_index.html
Comment (katedee) [reply content/en/join-our-team/_index.html:46]: Thanks for spotting this and the headings, and taking a look over the draft PR ✨ I'll make sure to correct it before it goes up for final reviews!

**Review by dj2 — COMMENTED** [review]
There is one header issue on the french template that I noticed. I think the `svg`s need the update that Calvin mentioned as well?

Otherwise, this all LGTM, but I'll leave it to someone on the website team to approve once the above are fixed.

File: content/fr/join-our-team/_index.html
Comment (katedee) [reply content/fr/join-our-team/_index.html:57]: ahh thanks for the catch! Thought I was careful but it's a bit tricky as the page is in two parts, essentially. Will get on that asap!

File: content/en/join-our-team/_index.html
Comment (omartehsin1) [inline content/en/join-our-team/_index.html:102]: Can we just remove the extra lines at the bottom. Otherwise LGTM!

**Comment by amazingphilippe** [comment]
This looks good to me! Keeping in mind the CMS changes and bigger more structural changes like for the page title.

One thing I realize snuck into the designs is this double-thick border, I think we should be making sure it looks consistent with the input border, or leave a gap in-between.
Capture d’écran, le 2021-07-29 à 14 02 47

Also worth pointing out that we need it to adapt to the mobile layout as well! here, a gap may be more robust and be easier to code. 
Capture d’écran, le 2021-07-29 à 14 00 54

**Review by dj2 — COMMENTED** [review]
LGTM. Will leave to someone from the website team to approve.

**Review by victoriachanimal — APPROVED** [review]
Woot! Thank you so much @katedee, this is awesome, and love that we kept the existing image in there. Content looks great. 

Will note / create some follow up Github issues for the remaining tasks (page title and menu updates, Strapi categories, @amazingphilippe's feedback about newsletter input border).

**Review by CalvinRodo — APPROVED** [review]
One final change for the SVGs that you know about other than that it's a +1 to me the page looks semantically good to me.

**Pull request merged**
